tools/build_api.py

```diff
@@ -457,6 +457,7 @@ def build_library(src_paths, build_path, target, toolchain_name,
         toolchain.copy_files(resources.headers, build_path, resources=resources)
         toolchain.copy_files(resources.objects, build_path, resources=resources)
         toolchain.copy_files(resources.libraries, build_path, resources=resources)
+        toolchain.copy_files(resources.json_files, build_path, resources=resources)
```

Comment:
What happens if the scanner finds more than one `mbed_lib.json` file? For example, the one for `core and the one for `rtos`. Are those copied in separate directories? It's important to keep all the `mbed_lib.json` files found by the scanner.